packages/koa-shopify-webhooks/src/register.ts

```diff
@@ -3,13 +3,14 @@ import {Method, Header} from '@shopify/network';
 import {WebhookHeader, Topic} from './types';
 
 export enum ApiVersion {
-  April19 = '2019-04',
   July19 = '2019-07',
   October19 = '2019-10',
   January20 = '2020-01',
```

Comment:
Let's also remove all the other unsupported versions:

```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/50832/118699360-70068e80-b7df-11eb-94e1-52921e279f97.png)